### PR TITLE
Fix some variable declarations due to > es6

### PR DESCRIPTION
**Pull request by potapovDim**
**Thanks for contributing to Selenium!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://github.com/SeleniumHQ/selenium/blob/master/CONTRIBUTING.md) guidelines.
Avoid large PRs, help reviewers by making them as simple and short as possible.

### Description
i want to propose you change all variables declaration due to > es6, remove functions what are used only in test files (and tests too). I want to ask you about some code style (jslint, eslint etc).

### Motivation and Context
I checked selenium code, and see that in current version engine is ' "node": ">= 
10.15.0"', from package.json. Many modules still using "var" declaration variables. 
Some functions required in tests files only.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [x] I have read the [contributing](https://github.com/SeleniumHQ/selenium/blob/master/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by potapovDim** [comment]
@corevo hey, can i ask about implementations of some code parts?

**Comment by corevo** [comment]
Sure

**Comment by potapovDim** [comment]
@corevo for example in one file i can see `} catch (ex) {` and ` } catch (e) {` is it normal ?

**Comment by corevo** [comment]
The requirement for the library is node 10, and it is not fully compliant with es6.

Be sure that whatever you change is compatible, the easiest way to check is to run the tests against node 10.

File: javascript/node/selenium-webdriver/README.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ also find the tests for selenium-webdriver informative.
 const {Builder, By, Key, until} = require('selenium-webdriver');
 
 (async function example() {
-  let driver = await new Builder().forBrowser('firefox').build();
+  const driver = new Builder().forBrowser('firefox').build();
```
Comment (corevo) [inline javascript/node/selenium-webdriver/README.md]: `build()` returns a Promise, therefore the `await` is required.

File: javascript/node/selenium-webdriver/package.json [hunk 2]
```diff
@@ -36,7 +36,7 @@
     "sinon": "^7.5.0"
   },
   "scripts": {
-    "test": "mocha -t 600000 --recursive test",
+    "test": "mocha -t 600000 --recursive ./**/*_test.js",
```
Comment (corevo) [inline javascript/node/selenium-webdriver/package.json]: What glob did you run to test that the same files are being pulled by mocha?

File: javascript/node/selenium-webdriver/README.md (hunk 1 above)
Comment (potapovDim) [reply javascript/node/selenium-webdriver/README.md]: of, got lost in implementation

File: javascript/node/selenium-webdriver/package.json (hunk 2 above)
Comment (potapovDim) [reply javascript/node/selenium-webdriver/package.json]: will back to initial

File: javascript/node/selenium-webdriver/lib/webdriver.js [hunk 3]
```diff
@@ -2021,9 +2020,8 @@ class WebElement {
    *     have been typed.
    */
   async sendKeys(...args) {
-    let keys = [];
-    (await Promise.all(args)).forEach(key => {
-      let type = typeof key;
+    let keys = (await Promise.all(args)).reduce((splitedKeys, key) => {
```
Comment (Harsha509) [inline javascript/node/selenium-webdriver/lib/webdriver.js:2037]: This is not going to work, it breaks the complete `sendKeys` method implementation. i would suggest to revert it to original!

File: javascript/node/selenium-webdriver/lib/logging.js [hunk 4]
```diff
@@ -646,8 +646,8 @@ class Preferences {
    */
   toJSON() {
     let json = {};
-    for (let key of this.prefs_.keys()) {
-      json[key] = this.prefs_.get(key).name;
+    for (let [key, value] of this.prefs_.entries()) {
+      json[key] = value;
```
Comment (Harsha509) [inline javascript/node/selenium-webdriver/lib/logging.js:650]: Invalid change, this breaks logging feature!

**Review by harsha509 — CHANGES_REQUESTED** [review]
Hi @potapovDim ,

Thanks for the PR.

The changes in this PR breaks few core features in JS bindings. I'd like to suggest to revert it to original (or) test the changes again with fixes update.

Regards,
Harsha

**Pull request closed without merging**